source/transactions-convenient-api/tests/README.md

```diff
@@ -41,8 +41,37 @@ If possible, drivers should implement these tests without requiring the test run
 the retry timeout. This might be done by internally modifying the timeout value used by `withTransaction` with some
 private API or using a mock timer.
 
+### Retry Backoff is Enforced
+
+Drivers should test that retries within `withTransaction` do not occur immediately. First, run transactions without
+backoff. To do so, configure the random number generator used for jitter to always return `0` -- this effectively
+disables backoff. Then, configure a fail point that forces 30 retries like so:
```

Comment:
I think this is supposed to be 13

Reply:
good catch! kinda went away with the test rewrite tho lol